changelogs/current.yaml

```diff
@@ -66,12 +66,12 @@ new_features:
     Added :ref:`trust_bundles
     <envoy_v3_api_field_extensions.transport_sockets.tls.v3.SPIFFECertValidatorConfig.trust_bundles>`
     to the SPIFFE certificate validator configuration. This field allows specifying a SPIFFE trust
-    bundle mapping as a DataSource. If both trust_bundles and trust_domains are specified,
-    trust_bundles takes precedence.
+    bundle mapping as a DataSource. If both ``trust_bundles`` and ``trust_domains`` are specified,
```

Comment:
not sure if ``DataSource`` should be literal - my rule of thumb (which sometimes is correct) is whether it would be translated

Reply:
I think it probably make sense to have it as ``DataSource``. WDYT?